packages/flutter_tools/test/general.shard/github_template_test.dart

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/base/io.dart';
+import 'package:flutter_tools/src/base/net.dart';
+import 'package:flutter_tools/src/reporting/github_template.dart';
+
+import '../src/common.dart';
+import '../src/context.dart';
+import '../src/testbed.dart';
+
+const String _kShortURL = 'https://www.example.com/short';
+
+void main() {
+  group('GitHub template creator', () {
+
+    testUsingContext('similar issues URL', () async {
+      final GitHubTemplateCreator creator = GitHubTemplateCreator();
+      expect(
+        await creator.toolCrashSimilarIssuesGitHubURL('this is a 100% error'),
+        _kShortURL);
```

Comment:
Formatting nit: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#if-you-have-a-newline-after-some-opening-punctuation-match-it-on-the-closing-punctuation

```dart
expect(
  await creator.toolCrashSimilarIssuesGitHubURL('this is a 100% error'),
  _kShortURL,
);
```